hack/generate-proto.sh

```diff
@@ -1,28 +1,27 @@
 #!/bin/bash
 set -eux -o pipefail
-
 go get k8s.io/code-generator/cmd/go-to-protobuf@v0.16.7-beta.0
 go get github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway@v1.12.1
 go get github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger@v1.12.1
 go get github.com/gogo/protobuf/protoc-gen-gogofast@v1.3.1
 go get github.com/gogo/protobuf/gogoproto@v1.3.1
-
 go install k8s.io/code-generator/cmd/go-to-protobuf
-
 go-to-protobuf \
     --go-header-file=./hack/custom-boilerplate.go.txt \
     --packages=github.com/argoproj/argo/pkg/apis/workflow/v1alpha1 \
-    --apimachinery-packages=+k8s.io/apimachinery/pkg/util/intstr,+k8s.io/apimachinery/pkg/api/resource,k8s.io/apimachinery/pkg/runtime/schema,+k8s.io/apimachinery/pkg/runtime,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/api/core/v1
-
+    --apimachinery-packages=+k8s.io/apimachinery/pkg/util/intstr,+k8s.io/apimachinery/pkg/api/resource,k8s.io/apimachinery/pkg/runtime/schema,+k8s.io/apimachinery/pkg/runtime,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/api/core/v1,k8s.io/api/policy/v1beta1  \
+    --proto-import ./vendor
```

Comment:
this is the wrong fix for this long term, as we should not be vendoring dependencies, but I think it is ok for today